spec/api/v1/events_spec.rb

```diff
@@ -50,6 +50,21 @@
         expect(api_event).to eq(serialized_event)
       end
     end
+
+    describe "GET /v1/events?filter[origin_system]" do
+      it "filters events by origin_system" do
+        FiveCalls_event = create(:event, title: 'eventA', browser_url: "www.eventA.com", start_date: Time.new(2019,01,06, 11, 25, 00), origin_system: "5calls")
```

Comment:
5Calls does advocacy campaigns (getting you to call an elected official or send an email) but events are CTA's that ask you to show up somewhere.  We should probably avoid making references to advocacy campaign sources inside tests for the events endpoint.